bazel/container_structure_test.bzl

```diff
@@ -26,11 +26,10 @@ CMD = """\
 
 readonly st=$(rlocation {st_path})
 readonly yq=$(rlocation {yq_path})
-readonly image=$(rlocation {image_path})
 
 # When the image points to a folder, we can read the index.json file inside
-if [[ -d "$image" ]]; then
-  readonly DIGEST=$("$yq" eval '.manifests[0].digest | sub(":"; "-")' "${{image}}/index.json")
+if [[ -d "{image_path}" ]]; then
+  readonly DIGEST=$("$yq" eval '.manifests[0].digest | sub(":"; "-")' "{image_path}/index.json")
```

Comment:
according to https://github.com/bazel-contrib/rules_oci/blob/770bedd27413eb093dd4123f1118876b61cb63da/oci/private/structure_test.bzl#L35  `--default-image-tag` needs to be added to args

Reply:
Yeah oops! Thanks

Reply:
unrelated question; the newest version of yq requires `-oj` to suppress the default output warning and `-r` to restore the output raw which was the default in the previous version.  can we also add those to yq call here to allow future versions of yq to be used?